conda-installation.md

```diff
@@ -35,10 +40,16 @@ your jobs. The other option is to install Miniconda inside each job.
 The first option is more efficient, especially for complex installations, 
 but there may be rare situations where installing with each job is better. 
 We recommend trying the pre-installation option first.
+If it doesn't work, discuss the second option with an RCF.
```

Comment:
"RCF" feels like jargon to me.  Is "facilitator" more widely understood by the average CHTC user?

Reply:
Probably.